components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3405,6 +3406,22 @@ public Group patchGroup(String groupId, String currentGroupName, Map<String, Lis
         return getGroup(groupId, requiredAttributes);
     }
 
+    /**
+     * Check whther the users in the set are already existing in the group.
+     *
+     * @param groupId                       Group Id.
+     * @param addedMemberIdsFromUserstore   Set of user ids.
+     */
+    private void removeAlreadyExistingUsersFromTheSet(String groupId, Set<String> addedMemberIdsFromUserstore) throws
```

Comment:
Please simplify the name may be as addedMemberIdsFromUserstore > userIdsSet

Reply:
Fixed with a2052ac42ef150002e6b6610c01bd529aab4b117